concourse/scripts/compile_gpdb.bash

```diff
@@ -59,6 +59,15 @@ function build_gpdb() {
 		make GPROOT=/usr/local PARALLEL_MAKE_OPTS=-j"$(nproc)" -s dist
 	fi
 	popd
+
+	if [ -n "${PYTHON3}" ]; then		
```

Comment:
So, if, in the absence of a second python, I set the PYTHON3 variable and run this script, then plpython will be compiled twice?

Reply:
Yes, "So Don't Hold It Like That." (c)

Reply:
Can we even build gpdb without python 2 with this script?

Reply:
Yes, just like it was before the change.  Just set PYTHON and don't set PYTHON3 env vars.